elixir/apps/api/lib/api/controllers/actor_controller.ex

```diff
@@ -0,0 +1,56 @@
+defmodule API.ActorController do
+  alias Domain.Actors
+  import API.ControllerHelpers
+  use API, :controller
+
+  action_fallback API.FallbackController
+
+  # List Actors
+  def index(conn, params) do
+    list_opts = params_to_list_opts(params)
+
+    with {:ok, actors, metadata} <- Actors.list_actors(conn.assigns.subject, list_opts) do
+      render(conn, :index, actors: actors, metadata: metadata)
+    end
+  end
+
+  # Show a specific Actor
+  def show(conn, %{"id" => id}) do
+    with {:ok, actor} <- Actors.fetch_actor_by_id(id, conn.assigns.subject) do
+      render(conn, :show, actor: actor)
+    end
+  end
+
+  # Create a new Actor
+  def create(conn, %{"actor" => params}) do
+    subject = conn.assigns.subject
+
+    with {:ok, account} <- Domain.Accounts.fetch_account_by_id(subject.account.id, subject),
```

Comment:
We already have `subject.account`, so we don't need to load it 2nd time

Reply:
Ah, yeah, that makes sense.  Will get this updated